appium/tests/specs/inbox/ReadEmailAfterRestartApp.spec.ts

```diff
@@ -0,0 +1,33 @@
+import {
+    SplashScreen,
+    CreateKeyScreen,
+    InboxScreen,
+    EmailScreen
+} from '../../screenobjects/all-screens';
+
+import {CommonData} from '../../data';
+
+describe('INBOX: ', () => {
+
+    it('user is able to see plain email without setting pass phrase after restart app', () => {
+
+        const senderEmail = CommonData.sender.email;
+        const emailSubject = 'Test 1';
+        const emailText = 'Test email';
+
+        const bundleId = CommonData.bundleId.id;
+
+        SplashScreen.login();
+        CreateKeyScreen.setPassPhrase();
+
+        InboxScreen.clickOnUserEmail(senderEmail);
```

Comment:
We'll have to be choosing emails in inbox by subject, there will be many more over time. Also, the subject will be more descriptive, eg in this case `plain email` instead of `Test email`.

Reply:
I am agree with you @tomholub , we need to send email via api before the test so we can pass dynamic subject and mail body to the test in future. I propose to create separate task for that.

Reply:
For now, you could do what you are doing - have the emails pre-arranged in the inbox instead of dynamically sending. But instead of going by sender, go by subject. Could you do that?